frontend/src/modules/scribble/stores/states/gamestates.js

```diff
@@ -19,11 +19,12 @@ class GameState {
 
 export class ChoosingWord extends GameState {
     static STATE = "ChoosingWord";
```

Comment:
This state is only active for the player choosing the word, so other players don't get this update. You can see it comes up as NaN for other players.
![image](https://user-images.githubusercontent.com/9893396/89132435-5f478f00-d4d1-11ea-96be-441a1577f777.png)